CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 4.2.1 (Under development)
 
+### App Center Crashes
+
+#### WPF/WinForms
```

Comment:
Is UWP good already because of sandbox?

Reply:
I didn't find that UWP stack trace contains username.